sphinx/domains/math.py

```diff
@@ -74,6 +74,8 @@ def note_equation(self, docname: str, labelid: str, location: Any = None) -> Non
                 labelid,
                 other,
                 location=location,
+                type='ref',
+                subtype='eq',
```

Comment:
```suggestion
                subtype='equation',
```

Reply:
Ah, I was torn between the two and went with `eq` because it matched the role name